jujugui/static/gui/src/app/components/machine-view/machine-view.js

```diff
@@ -19,40 +19,42 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 'use strict';
 
 class MachineView extends React.Component {
-  constructor(props) {
-    super(props);
+  constructor() {
+    super();
```

Comment:
Shouldn't we always call super with props? If that's not the case, could you please ad a comment?